src/modules/access-wallet/software/handlers/web3wallet/errorHandler.js

```diff
@@ -9,7 +9,9 @@ const ERRORS = {
     'Returned error: Error: MetaMask Tx Signature: User denied transaction signature.',
   'User rejected action!': 'User rejected action!',
   "Can't sign messages from a burner account!":
-    "Can't sign messages from a burner account!"
+    "Can't sign messages from a burner account!",
+  "Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.":
```

Comment:
This is better set on the global errors since this is a web3 issue that could happen with any wallet

Reply:
Fixed!